src/BehaviorsSDKManaged/.editorconfig

```diff
@@ -0,0 +1,238 @@
+dotnet_naming_rule.static_readonly_fields_must_be_pascal_case.severity            = silent
+dotnet_naming_rule.static_readonly_fields_must_be_pascal_case.symbols             = static_readonly_fields
+dotnet_naming_rule.static_readonly_fields_must_be_pascal_case.style               = pascal_case
+# Private readonly fields must be camelCase
+dotnet_naming_rule.private_readonly_fields_must_be_camel_case.severity            = silent
+dotnet_naming_rule.private_readonly_fields_must_be_camel_case.symbols             = private_readonly_fields
+dotnet_naming_rule.private_readonly_fields_must_be_camel_case.style               = camel_case
+# Public and internal fields must be PascalCase
+dotnet_naming_rule.public_internal_protected_fields_must_be_pascal_case.severity  = silent
+dotnet_naming_rule.public_internal_protected_fields_must_be_pascal_case.symbols   = public_internal_protected_fields
+dotnet_naming_rule.public_internal_protected_fields_must_be_pascal_case.style     = pascal_case
+# Private and protected fields must be camelCase
+dotnet_naming_rule.private_fields_must_be_camel_case.severity                     = silent
+dotnet_naming_rule.private_fields_must_be_camel_case.symbols                      = private_protected_fields
+dotnet_naming_rule.private_fields_must_be_camel_case.style                        = prefix_private_field_with_underscore
+# Public members must be capitalized
+dotnet_naming_rule.public_members_must_be_capitalized.severity                    = silent
+dotnet_naming_rule.public_members_must_be_capitalized.symbols                     = public_symbols
+dotnet_naming_rule.public_members_must_be_capitalized.style                       = first_upper
+# Parameters must be camelCase
+dotnet_naming_rule.parameters_must_be_camel_case.severity                         = silent
+dotnet_naming_rule.parameters_must_be_camel_case.symbols                          = parameters
+dotnet_naming_rule.parameters_must_be_camel_case.style                            = camel_case
+# Class, struct, enum and delegates must be PascalCase
+dotnet_naming_rule.non_interface_types_must_be_pascal_case.severity               = silent
+dotnet_naming_rule.non_interface_types_must_be_pascal_case.symbols                = non_interface_types
+dotnet_naming_rule.non_interface_types_must_be_pascal_case.style                  = pascal_case
+# Interfaces must be PascalCase and start with an 'I'
+dotnet_naming_rule.interface_types_must_be_prefixed_with_i.severity               = silent
+dotnet_naming_rule.interface_types_must_be_prefixed_with_i.symbols                = interface_types
+dotnet_naming_rule.interface_types_must_be_prefixed_with_i.style                  = prefix_interface_interface_with_i
+# prefix_private_field_with_underscore - Private fields must be prefixed with _
+dotnet_naming_style.prefix_private_field_with_underscore.capitalization           = camel_case
+dotnet_naming_style.prefix_private_field_with_underscore.required_prefix          = _
```

Comment:
This goes directly against our [coding guidelines](https://github.com/Microsoft/XamlBehaviors/wiki/Contributing-Guidelines). Unfortunately I don't see a way to say that is should NOT start with _ or m_.

Reply:
I'd rather see the underscore prefix than using `this.`. It can be confusing when variables are not underscore prefixed. 
```
private int myValue;

private void Foo(int myValue)
{
    // wait, now we have two myValue variables!!?!?
}
```

Reply:
We could easily change. I would be happy to go through the repo to fix up private variables